patch-package
--- a/node_modules/react-virtualized-tree/es/Tree.js
+++ b/node_modules/react-virtualized-tree/es/Tree.js
@@ -38,7 +38,7 @@ var Tree = function (_React$Component) {
 
       return React.createElement(
         'div',
-        { key: key, className: 'tree-node', style: _extends({}, style, { marginLeft: node.deepness * nodeMarginLeft }) },
+        { key: key, className: 'tree-node', style: _extends({}, style, { paddingLeft: node.deepness * nodeMarginLeft }) },
         React.createElement(NodeRenderer, { node: node, onChange: _this.props.onChange, measure: measure })
       );
     }, _this.measureRowRenderer = function (nodes) {
--- a/node_modules/react-virtualized-tree/lib/Tree.js
+++ b/node_modules/react-virtualized-tree/lib/Tree.js
@@ -51,7 +51,7 @@ var Tree = function (_React$Component) {
 
       return _react2.default.createElement(
         'div',
-        { key: key, className: 'tree-node', style: _extends({}, style, { marginLeft: node.deepness * nodeMarginLeft }) },
+        { key: key, className: 'tree-node', style: _extends({}, style, { paddingLeft: node.deepness * nodeMarginLeft }) },
         _react2.default.createElement(NodeRenderer, { node: node, onChange: _this.props.onChange, measure: measure })
       );
     }, _this.measureRowRenderer = function (nodes) {
